gradle.properties

```diff
@@ -6,3 +6,5 @@ android.enableR8.fullMode=true
 android.useAndroidX=true
 
 agpVersion=7.4.0
+
+org.gradle.jvmargs=-Xmx2048m
```

Comment:
dont write it here.

Reply:
I encounter the same problem as this [StackOverflow question](https://stackoverflow.com/a/70756876/7870953).
Subproject setting is not possible, so I set it only for my local machine.